### PR TITLE
Fix/travis

### PR DESCRIPTION
**Pull request by akovalyov**

### CODE REVIEW

**Comment by umpirsky** [comment]
:+1:

File: .travis.yml [hunk 1]
```diff
@@ -1,7 +1,12 @@
 language: php
 
+sudo: false
+
+cache:
+  directories:
+    - $HOME/.composer/cache
```
Comment (stof) [inline .travis.yml]: I suggest caching only the ``files`` subfolder. See https://github.com/puli/manager/pull/9#discussion_r40499480 for the explanation

File: .travis.yml [hunk 2]
```diff
@@ -10,10 +15,12 @@ php:
 matrix:
     allow_failures:
         - php: hhvm
+    fast_finish: true
 
 before_script:
-  - ./bin/configure_test_env.sh
-  - composer self-update && composer install --dev --prefer-source --no-interaction
+  - echo "extension = mongo.so" >> ~/.phpenv/versions/$(phpenv version-name)/etc/php.ini
```
Comment (stof) [inline .travis.yml]: this should be skipped on HHVM (and PHP 7 but it is not yet enabled) as mongo.so is not available there

File: .travis.yml [hunk 3]
```diff
@@ -10,10 +15,12 @@ php:
...
     allow_failures:
         - php: hhvm
+    fast_finish: true
 
 before_script:
-  - ./bin/configure_test_env.sh
-  - composer self-update && composer install --dev --prefer-source --no-interaction
+  - echo "extension = mongo.so" >> ~/.phpenv/versions/$(phpenv version-name)/etc/php.ini
+  - cp tests/Gaufrette/Functional/adapters/DoctrineDbal.php.dist tests/Gaufrette/Functional/adapters/DoctrineDbal.php -f
+  - composer self-update && composer install --dev --prefer-source --no-interaction --ignore-platform-reqs
```
Comment (stof) [inline .travis.yml]: ``--dev``is a deprecated no-op option (well, actually, it does something: displaying a deprecation warning)

**Comment by akovalyov** [comment]
@stof thank you for your valuable comments

**Pull request merged**